src/main/java/org/ays/auth/controller/AysUserController.java

```diff
@@ -152,6 +152,24 @@ public AysResponse<Void> activate(@PathVariable @UUID final String id) {
     }
 
 
+    /**
+     * PATCH /user/{id}/passivate : Passivates (deactivates) a user account with the given ID.
+     * <p>
+     * This endpoint is protected and requires the caller to have the authority
+     * 'user:update'. The user ID must be a valid UUID.
+     * </p>
+     *
+     * @param id The UUID of the user to be passivated.
+     * @return An {@link AysResponse} indicating the success of the operation.
+     */
+    @PatchMapping("user/{id}/passivate")
```

Comment:
Bir üstteki aktifleştirme servisinde de gözden kaçmış sanırım. Orayı da aşağıdaki şekilde düzeltebiliriz;
```suggestion
    @PatchMapping("/user/{id}/passivate")
```